pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -80,6 +87,10 @@ def distributed_sampler_kwargs(self):
         distributed_sampler_kwargs = dict(num_replicas=(self.num_nodes * self.num_processes), rank=self.global_rank)
         return distributed_sampler_kwargs
 
+    @property
+    def is_single_process_single_device(self):
+        return True
+
```

Comment:
would this be called anywhere outside this class? 
```suggestion
    @property
    def is_single_process_single_device(self) -> bool:
        return True

```

Reply:
I do not think this will be called outside class.

Reply:
```suggestion
    @property
    def _is_single_process_single_device(self):
        return True

```